simpeg/maps/__init__.py

```diff
@@ -0,0 +1,35 @@
+from ._base import (
+    ComboMap,
+    IdentityMap,
+    LinearMap,
+    Projection,
+    SelfConsistentEffectiveMedium,
+    SphericalSystem,
+    SumMap,
+    Wires,
+)
+from ._clustering import PolynomialPetroClusterMap
+from ._injection import Mesh2Mesh, InjectActiveCells
+from ._mesh_agnostic import (
```

Comment:
rather than `mesh agnostic`, what do you think of `property_transforms` (e.g. these are one-to-one mappings that map from a property from one space to another)?